app/assets/stylesheets/tile.css

```diff
@@ -0,0 +1,49 @@
+@media screen and (min-width : 800px) {
+    .tile-wrapper {
+        display: grid;
```

Comment:
How do we handle browser support? See https://caniuse.com/#feat=css-grid
Do we have a fallback to flexbox or something similar?

Reply:
Grid is not compatible with Opera Mini and UC Browser for Android. All major browsers support it. So I guess we're fine :).